test/integration/cloud_storage_test.go

```diff
@@ -0,0 +1,58 @@
+//go:build integration
+
+package cloud_storage
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/gruntwork-io/terratest/modules/random"
+
+	// import the blueprints test framework modules for testing and assertions
```

Comment:
Please delete all comments except the one that I've commented (#L16). 
These provide no additional information and your naming and the API makes it clear what's going on 👍🏻